external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -138,6 +121,31 @@
     <PackageToInclude Include="CommandLineParser.NS20" />
   </ItemGroup>
 
+  <!-- Test SDK packages. -->
+  <ItemGroup Condition="'$(IncludeVSTestReferences)' == 'true'">
```

Comment:
Is IncludeVSTestReferences set to true for Xunit.Runtime.depproj? I doubt it will be set to true since we set it here: https://github.com/dotnet/buildtools/blob/0fa455c550abb4f49f4830a7f2fa78609f373fd0/src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets#L76

And this project is executed when syncing and restoring packages and it will not fall into the IsInvokableTestProject category

Reply:
Oh right, thanks!